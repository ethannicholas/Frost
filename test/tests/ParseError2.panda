method main() {
    Console.printLine(extras.Broken())
}