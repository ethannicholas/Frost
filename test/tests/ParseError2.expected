Broken.panda:8:5: error: expected a declaration, but found 'Broken'
ParseError2.panda:2:23: error: unknown identifier
2 errors
